### DISCUSSION TITLE
Import sql into supabase

### DISCUSSION THREAD

**Discussion by sunillarsson** [supabase/supabase#550] [Questions] [answered]
Is there a way to import large-ish postgres tables into supabase? My table is about 6k rows and 18 columns. I've tried the browser sql-editor, but it seems to max out at about 200 rows in this case.

**Comment by silentworks** [comment, marked as answer]
You can connect to supabase with a desktop SQL client. I'm using TablePlus and was able to connect. You can find your database connection details under `Settings > Database`.

**Reply by sunillarsson** [reply]
Ah, that worked perfectly. Thank you! :)
